packages/pa_ppx_migrate/pa_ppx_migrate.0.13/opam

```diff
@@ -0,0 +1,49 @@
+
+synopsis: "A PPX Rewriter for Migrating AST types (written using Camlp5)"
+description:
+"""
+This is a PPX Rewriter for generating "migrations" like those written
+by-hand (with some automated support) in "ocaml-migrate-parsetree".
+The goal here is that the input to the automated tool is the minimum
+possible, with no need for human massaging of output from the tool.
+
+There are two examples: a small one (in a unit-test) and a full set of
+migrations from Ocaml's AST 4.02 all the way up to 5.2.0, with all the
+intermediate ASTs, and migrations forward as well as backward.
+
+"""
+opam-version: "2.0"
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+authors: ["Chet Murthy"]
+homepage: "https://github.com/camlp5/pa_ppx_migrate"
+license: "BSD-3-Clause"
+bug-reports: "https://github.com/camlp5/pa_ppx_migrate/issues"
+dev-repo: "git+https://github.com/camlp5/pa_ppx_migrate.git"
+doc: "https://github.com/camlp5/pa_ppx_migrate/doc"
+
+depends: [
+  "ocaml"       { >= "4.10.0" }
+  "cppo" { >= "1.6.9" }
+  "camlp5-buildscripts" { >= "0.02" }
+  "camlp5"      { >= "8.03.00" }
+  "pa_ppx"      { >= "0.15" }
+  "not-ocamlfind" { >= "0.10" }
+  "ounit" { >= "2.2.7" & with-test}
+  "fmt"
+  "bos" { >= "0.2.0" }
+]
+conflicts: [
```

Comment:
I have the same comment about the conflicts as in the other PR. We should move them to the respective packages if this one is an explicit dependency